### ISSUE TITLE
maximumFileSizeToCacheInBytes

### ISSUE THREAD

**Issue by Sebah7** [chas-academy/u09-Kind-Earth-Skincare-Sebah7#55]
### <span aria-hidden="true">❌</span> Deploy Preview for *kindearthskincare* failed. **[Why did it fail? →](https://app.netlify.com/sites/kindearthskincare/deploys/67129ef3929edb0008c874f8?deploy_diagnostics=true)**

|  Name | Link |
|:-:|------------------------|
|<span aria-hidden="true">🔨</span> Latest commit | 172c8474d883367b93a6052327bab2bc37de6abc |
|<span aria-hidden="true">🔍</span> Latest deploy log | https://app.netlify.com/sites/kindearthskincare/deploys/67129ef3929edb0008c874f8 |

_Originally posted by @netlify[bot] in https://github.com/chas-academy/u09-Kind-Earth-Skincare-Sebah7/issues/54#issuecomment-2422956269_

**Comment by Sebah7** [comment]
Added maximumFileSizeToCacheInBytes: 3000000 to vite and some more image formats
